db/system_keyspace.cc

```diff
@@ -263,6 +263,7 @@ schema_ptr built_indexes() {
                 {"scylla_cpu_sharding_algorithm", utf8_type},
                 {"scylla_nr_shards", int32_type},
                 {"scylla_msb_ignore", int32_type},
+                {"cdc_streams_timestamp", timestamp_type},
```

Comment:
Need a comment explaining what this is.

Does this affect downgrades? We must make sure it isn't populated until the cluster agrees all nodes support this feature (at which point downgrades become impossible).

Reply:
Why not just tell the user to make a backup before upgrading?

Anyway, this value won't be populated unless the user enables cdc in configuration. And we don't officially support upgrading from previous versions of cdc (I heard that's one of the purposes of the "experimental" flag), so the user in theory couldn't have enabled cdc before upgrading.
So we can say: "don't enable CDC before you finish upgrading the whole cluster, or you won't be able to downgrade in the middle". Is that acceptable?

Reply:
We do tell the user to make a backup, but that's for recovering from a disaster, not as part of normal upgrade/downgrade. Recovering from a backup involves downtime.

While we can tell the user not to enable cdc, it's better to have the code enforce it. Not all users read the instructions diligently.

Reply:
I think it'll be easiest to make a separate table for CDC local info, so if the user downgrades, he'll just be left with a garbage table (populated or not) with a single timestamp, which he can then remove.

Reply:
I made a separate local table.